.github/workflows/build-test.yml

```diff
@@ -135,140 +112,97 @@ jobs:
 
   test-gradle:
     name: Gradle tests
-    runs-on: ubuntu-18.04
-    strategy:
-      fail-fast: false
+    runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
 
-      - name: Checkout GitHub merge
```

Comment:
Does this mean we're now testing the branch again, rather than the merge of the branch to the target branch?

Reply:
No, default is merge commit for `actions/checkout`. See https://github.com/actions/checkout#checkout-pull-request-head-commit-instead-of-merge-commit

Reply:
Ah, I was confusing GA and CircleCI then I guess :) :+1: